017-FrontDoor/Coach/Solution02.md

```diff
@@ -0,0 +1,36 @@
+### Notes for Challenge 2 - Provision a Web Application Firewall (WAF)
+
+ [< Previous Challenge [1]](./Solution01.md) - **[Home](./README.md)** - [Next Challenge [3] >](./Solution03.md)
+
+
+Only from a time perspective does the challenge specifies to mark the WAF Policy into Prevention mode from the start.  In normal circumstances, this is **SUPER DANGEROUS**.  WAF Rules cover various patterns that over the years and years web traffic has be subverted.  Sometimes the approaches of the past generated traffic patterns that are actually "OK" for some websites.  Watching in Detection mode is extremely important to be able to evaluate the effectiveness and/or over aggressiveness of each of the rules.
+
+A really popular strategy is that, after an initial amount of traffic goes thru a rule set, is to set rules in "Log" mode and flip the Policy into Prevention.  This allows a more granular approach to enable WAF rules.  
+
+We running `w3af` to generate simulate bad traffic against Front Door to be able to show how fo find malicious traffic.  **REMEMBER** It takes times (5 minutes or so) after a Diagnostics Settings being created for it to be logging.  And there is another delay have logs appear after the hits.  It's really important that people see logs in the Challenge 1 before starting Challenge 2.  The goal isn't to learn how to use `w3af` :).  Help them out as much as needed to get it to run.
+
+We use https://tools.keycdn.com/performance to show the geo-blocking is effective.
+
+## Links
+- [Create a Front Door WAF Policy](https://docs.microsoft.com/en-us/azure/web-application-firewall/afds/waf-front-door-create-portal)
+  
+## Solution Scripts (PowerShell with AZ CLI)
+
+#### Create WAF Policy with Default Rule Set and Only allow US
+
+- `az network front-door waf-policy create --name wwwWAFPolicy --resource-group rg-MyResourceGroup --mode Detection`
```

Comment:
I would change this section to be a full code block.... by doing this:
1. delete the ` surrounding each of the lines of code
2. delete the '- ' from each line so they are no longer bullets
3. add a newline between each line
4. surround the whole block with ``` above and below the block